Task list component guidance and updated pattern page (#1994)
alphagov/govuk-design-system: src/patterns/complete-multiple-tasks/index.md

Make it clear to users which tasks they’ve completed and which still need their attention, by labelling them using statuses.

{{ example({ group: "patterns", item: "complete-multiple-tasks", example: "task-list-statuses", html: true, nunjucks: true, open: false }) }}

Statuses should be helpful to users. The more you add, the harder it is for users to remember them. Start with the smallest number of different statuses you think might work, for example ‘Completed’ and ‘Incomplete’, then add more if your user research shows there’s a need for them.

Once the user has completed the task, the status should show as ‘Completed’ and be black text with no background colour. This will draw more attention to tasks that require action.

#### Tasks that are in progress

You may find you need additional statuses if your user research shows that users want to be able to distinguish between the tasks they haven’t started at all, and those they’ve started but not completed.

In this instance, instead of ‘Incomplete’, you may want to use ‘Not yet started’ to show which tasks they are yet to start. You should then use ‘In progress’ for tasks they have started but are yet to complete.

> ‘In progress’ is dark blue text on a blue background colour. ‘Not yet started’ is dark blue on a light blue background colour.

Reviewer: This paragraph:

* doesn’t accurately describe the status colours shown
* doesn’t align with tag colour names
* is confusing as it mentions both text and background colours; elsewhere on this page only the background colour is referred to
* uses a different order of statuses to the preceding paragraph

```suggestion
‘Not yet started’ uses a blue background colour. ‘In progress’ uses a light blue background colour.
```

Reviewer: Perfect, thanks!
